### DISCUSSION TITLE
how to set check point as our need like after 100 EPOCHS of 275 EPOCH ?and how restore those check point???

### DISCUSSION THREAD

**Discussion by omkarade** [coqui-ai/TTS#1748] [General Q&A]
how to set check point as our need like after 100 EPOCHS of 275 EPOCH ?and how restore those check point???

**Comment by p0p4k** [comment]
Sir, I would suggest you to go through the train.py file. And understand the different parameters.

**Reply by omkarade** [reply]
can you please share me link of train.py file.

**Reply by p0p4k** [reply]
check the train files inside the model of your choice. 
https://github.com/coqui-ai/TTS/tree/dev/recipes
You seem to be very new to python and TTS, this is not a support forum, its a discussion area for open-source TTS. Please try to use Google or GitHub search bar and find your solutions first. Also, try to read the official documentation https://tts.readthedocs.io/en/latest/ . So that you can understand the basics first!
If you need more help, message me on gitter.im , I might help you with basic stuff.
Thanks. Good luck.

**Reply by omkarade** [reply]
Thanks!
please give me your gitter.im link.

**Reply by p0p4k** [reply]
It is same username as GitHub -> p0p4k

**Comment by thorstenMueller** [comment]
Hello @omkarade ,
some small general advices for asking questions here.

- Maybe just open one question instead of asking every minor aspect in a separate discussion.
- Choose a good title for discussion that helps understand what your general problem is.
- One question mark in title is sufficient - people will understand that it's a question.
- Please share ...
  - What your final goal is
  - What is your environment/setup
  - What have you tried so far
  - Did exceptions occur (which one)
  - Which part of documentation/code have you read to figure out a solution?
  - Share links to config and/or dataset if it's publically visible

This will it make easier to help understand your problems and find solutions to fix it 🙂
